package.json

```diff
@@ -1,116 +1,26 @@
-    "build": "babel-node bin/build.js && npm run open:dist",
-    "test": "jest --config config/jest.json",
-    "test:cover": "npm run test -- --coverage ",
-    "test:watch": "jest --config config/jest.json --watch",
-    "open:cover": "npm run test:cover && opn ./coverage/lcov-report/index.html",
-    "analyze-bundle": "babel-node ./bin/analyzeBundle.js"
-  },
-  "author": "Abhinav Mishra",
-  "license": "MIT",
-  "dependencies": {
-    "object-assign": "4.1.1",
-    "react": "16.2.0",
-    "react-dom": "16.2.0",
-    "react-redux": "5.0.6",
-    "react-router-dom": "4.2.2",
-    "react-router-redux": "5.0.0-alpha.6",
-    "redux": "3.7.2",
-    "redux-thunk": "2.2.0",
-    "webpack-merge": "4.1.1"
-  },
-  "devDependencies": {
-    "@nimbl3/eslint-config-nimbl3": "2.0.1",
-    "autoprefixer": "7.2.3",
-    "babel-cli": "6.26.0",
-    "babel-core": "6.26.0",
-    "babel-eslint": "8.0.3",
-    "babel-jest": "22.0.3",
-    "babel-loader": "7.1.2",
-    "babel-plugin-dynamic-import-node": "1.2.0",
-    "babel-plugin-transform-react-constant-elements": "6.23.0",
-    "babel-plugin-transform-react-remove-prop-types": "0.4.12",
-    "babel-polyfill": "6.26.0",
-    "babel-preset-env": "1.6.1",
-    "babel-preset-react": "6.24.1",
-    "babel-preset-react-hmre": "1.1.1",
-    "babel-preset-stage-1": "6.24.1",
-    "browser-sync": "2.19.0",
-    "chalk": "2.3.0",
-    "concurrently": "3.5.1",
-    "connect-history-api-fallback": "1.5.0",
-    "coveralls": "3.0.0",
-    "css-loader": "0.28.7",
-    "enzyme": "3.2.0",
-    "enzyme-adapter-react-16": "1.1.1",
-    "eslint": "4.13.1",
-    "eslint-plugin-import": "2.8.0",
-    "eslint-plugin-react": "7.5.1",
-    "eslint-watch": "3.1.3",
-    "extract-text-webpack-plugin": "3.0.2",
-    "file-loader": "1.1.6",
-    "glob": "7.1.2",
-    "history": "4.7.2",
-    "html-webpack-plugin": "2.30.1",
-    "identity-obj-proxy": "3.0.0",
-    "jest": "22.0.3",
-    "jest-cli": "22.0.3",
-    "json-loader": "0.5.7",
-    "less": "2.7.3",
-    "less-loader": "4.0.5",
-    "mockdate": "2.0.2",
-    "node-sass": "4.7.2",
-    "opn-cli": "3.1.0",
-    "postcss-loader": "2.0.9",
-    "prompt": "1.0.0",
-    "prop-types": "15.6.0",
-    "raf": "3.4.0",
-    "react-hot-loader": "3.1.3",
-    "react-test-renderer": "16.2.0",
-    "redux-immutable-state-invariant": "2.1.0",
-    "replace": "0.3.0",
-    "rimraf": "2.6.2",
-    "sass-loader": "6.0.6",
-    "style-loader": "0.19.1",
-    "url-loader": "0.6.2",
-    "webpack": "3.10.0",
-    "webpack-bundle-analyzer": "2.9.1",
-    "webpack-dev-middleware": "2.0.2",
-    "webpack-hot-middleware": "2.21.0",
-    "webpack-md5-hash": "0.0.6"
-  },
+  "name": "cra-template-nimble",
+  "version": "1.1.2",
   "keywords": [
     "react",
-    "reactjs",
-    "react-router",
-    "hot",
-    "reload",
-    "hmr",
-    "live",
-    "edit",
-    "webpack",
-    "redux",
-    "boilerplate",
-    "starter"
+    "create-react-app",
+    "template",
+    "typescript"
```

Comment:
Should we add a "nimble" keyword as well?

Reply:
✅  added in [7f1c1ce](https://github.com/nimblehq/react-templates/pull/12/commits/7f1c1ce3ea1a3d6ec6996ee3f76b9ac9a70d01a5)